64_multi_agent_llm_trading/python/backtest.py

```
@@ -0,0 +1,572 @@
+            lookback: Number of periods for analysis lookback
+            step: Rebalancing frequency (1 = every day)
+            context: Additional context passed to agents
+
+        Returns:
+            BacktestResult with performance metrics
+        """
+        self.reset()
+        context = context or {}
+
+        if len(data) < lookback + 10:
+            raise ValueError(f"Insufficient data: need at least {lookback + 10} periods")
```

Ruff (0.14.13)

Avoid specifying long messages outside the exception class

(TRY003)